README.md

```diff
@@ -70,7 +70,9 @@ An admin is available at [http://localhost:8000/admin/](http://localhost:8000/ad
 Run `python manage.py test --keepdb` to run the tests.  The `--keepdb` is important so Django doesn't try to destroy the test database.
 
 ### Frontend configuration
-From project root, `cd repo_health/index/static`.  Run `npm install`.  After successful install, [http://localhost:8000](http://localhost:8000) should display the Hello World page.
+From project root, `cd repo_health/index/static`.  
+
+Run `npm start` and go to [http://localhost:3000](http://localhost:3000) to view the application.
 
 To run ui tests, run `npm test` in the `repo_health/index/static` folder.
```

Comment:
I'll update it for this PR, and let you know 👍 . It should be in when this goes in.